playbooks/verify.yml

```diff
@@ -23,3 +23,12 @@
   # Run sanity test
   - import_role:
       name: verify
+
+  - name: Verify fingerprint is correct
+    raw: >-
+        touch {{ zos_uss_user_profile }} && \
+        . {{ zos_uss_user_profile }} \
+        {{ zowe_environment_variable_overrides | default('') }} && \
+        cd "{{ zowe_root_dir }}" && \
+        bin/zowe-verify-authenticity.sh -l "{{ zowe_install_logs_dir }}"
```

Comment:
This is duplicated since you also have defined in `playbooks/roles/verify/tasks/main.yml`.

```
- import_role:
      name: verify
```
will call what you defined in `playbooks/roles/verify/tasks/main.yml`.

Reply:
only mocha used now.